package.json

```diff
@@ -6,7 +6,7 @@
     "build": "rm -f ./public/build/* && rollup -c",
     "dev": "rollup -c -w",
     "start": "sirv public --no-clear",
-    "deploy": "npm run build && rsync -ar -e 'ssh -p 55022' --delete --stats --progress ./public/* developer@www.digitale-gesellschaft.ch:~/wwwtest/auskunftsbegehren"
+    "deploy": "npm run build && rsync -ar -e 'ssh -p 55022' --delete --stats --progress --exclude ./public/data.json ./public/* developer@www.digitale-gesellschaft.ch:~/wwwtest/auskunftsbegehren"
```

Comment:
syncing the data.json is currently part of the process for the test deployment, so it must not be excluded.